console-subscriber/src/builder.rs

```diff
@@ -268,6 +274,24 @@ impl Builder {
         Self { self_trace, ..self }
     }
 
+    #[cfg(feature = "grpc-web")]
```

Comment:
The `#[cfg]` line should be below the doc comments. It all works the same, but it is a bit easier to reason about.

Reply:
Updated.